### PR TITLE
NAS RL Suggestion for v1alpha2

### PR DESCRIPTION
**Pull request by andreyvelich**
This is the NAS RL Suggestion implementation for v1alpha2. 
I am still working on it, fixing some problems, but we can start to review it.

/assign @johnugeorge @hougangliu @gaocegege 
/cc @richardsliu

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/613)

### CODE REVIEW

**Comment by andreyvelich** [comment]
I have done all the work and fixed all errors.
I ran this Suggestion on my cluster successfully.

File: cmd/suggestion/nasrl/v1alpha2/main.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import grpc
+from concurrent import futures
+
+import time
+
+from pkg.api.v1alpha2.python import api_pb2_grpc
+from pkg.suggestion.v1alpha2.nasrl_service import NasrlService
+from logging import getLogger, StreamHandler, INFO, DEBUG
```
Comment (gaocegege) [inline cmd/suggestion/nasrl/v1alpha2/main.py]: Do we need this here?

**Comment by andreyvelich** [comment]
> /lgtm
> 
> Do we have an example for NASRL in v1alpha2?

Yea, here: https://github.com/kubeflow/katib/blob/master/examples/v1alpha2/nasjob-example-RL.yaml

File: cmd/suggestion/nasrl/v1alpha2/main.py (hunk 1 above)
Comment (andreyvelich) [reply cmd/suggestion/nasrl/v1alpha2/main.py]: Thanks, remove.

**Pull request merged**